raaghu-elements/src/rds-dropdown-list/rds-dropdown-list.tsx

```
@@ -1,11 +1,11 @@
 import React, { useState, useEffect, useRef } from "react";
 import RdsIcon from "../rds-icon";
 import RdsBadge from "../rds-badge";
-import RdsCheckbox from "../rds-checkbox";
 import "./rds-dropdown-list.css";
 import Tooltip from "../rds-tooltip/rds-tooltip";
```

ESLint

[error] 'Tooltip' is defined but never used.

(@typescript-eslint/no-unused-vars)